x/ecocredit/server/core/send.go

```diff
@@ -151,7 +151,7 @@ func (k Keeper) sendEcocredits(ctx context.Context, credit *core.MsgSend_SendCre
 			Retirer:    to.String(),
 			BatchDenom: credit.BatchDenom,
 			Amount:     sendAmtRetired.String(),
-			Location:   credit.RetirementLocation,
+			Location:   credit.RetirementJurisdiction,
```

Comment:
And here for `EventRetire`.

Reply:
ah, looks looks like this was using an outdated event too! good catch 👍🏻

Reply:
Nice. At least one of my comments was relevant.